WcaOnRails/app/views/teams/_team_member_fields.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="nested-fields">
```

Comment:
Can this file get removed now thanks to simple_form's support for nested fields?